internal/backend/oauth/webhook.go

```diff
@@ -25,8 +25,12 @@ type svc struct {
 
 func InitWebhook(l *zap.Logger, muxes *muxes.Muxes, c sdkservices.Services) {
 	s := &svc{logger: l, svcs: c}
-	muxes.Auth.HandleFunc("GET /oauth/start/{intg}", s.start)
-	muxes.NoAuth.HandleFunc("GET /oauth/redirect/{intg}", s.exchange)
+
+	muxes.Main.Auth.HandleFunc("GET /oauth/start/{intg}", s.start)
+	muxes.Main.NoAuth.HandleFunc("GET /oauth/redirect/{intg}", s.exchange)
+
+	muxes.Aux.Auth.HandleFunc("GET /oauth/start/{intg}", s.start)
+	muxes.Aux.NoAuth.HandleFunc("GET /oauth/redirect/{intg}", s.exchange)
```

Comment:
See my comment above about the excessive complexity of mutex trees. This specifically is an example of a foot gun: which is the right mutex to use, given that both support the same URLs?

Please document what it means/does that both ports support identical URLs.